ui/src/components/Time.tsx

```diff
@@ -1,23 +1,27 @@
-import { useMemo } from "preact/hooks";
+import { useState, useEffect, useMemo, useCallback } from "preact/hooks";
 import TimeAgo, { Formatter, ReactTimeagoProps } from "react-timeago";
 
 interface TimeProps extends ReactTimeagoProps {
   date: string | number;
 }
 
 const formatter: Formatter = (value, unit, suffix, ms, next) => {
-  if (unit === "second") {
-    return "less then a minute ago";
-  }
-
   return next?.(value, unit, suffix, ms);
 };
 
 /**
  * TODO Add tooltip on hover with full date.
  */
 export const Time = ({ date }: TimeProps) => {
+
   const tooltip = useMemo(() => new Date(date).toLocaleString(), [date]);
 
-  return <TimeAgo date={date} formatter={formatter} title={tooltip} />;
+  // Refresh each second such that this component re-renders times appropriately.
+  const [time, setTime] = useState(new Date().valueOf());
+  useEffect(() => {
+    const interval = setInterval(() => { setTime(new Date().valueOf()) }, 1000);
+    return () => { clearInterval(interval); };
+  }, []);
+
+  return <TimeAgo date={date} formatter={formatter} title={tooltip} key={time} />;
```

Comment:
[react-timeago](https://www.npmjs.com/package/react-timeago) should already do this

Reply:
It doesn't, for some reason :/